packages/dappmanager/test/modules/compose/labelDb.test.ts

```diff
@@ -109,7 +109,7 @@ describe("Parse and validate manifest labels to be used in the compose", () => {
       "dappnode.dnp.dependencies": '{"oneDependency": "dependency-name"}',
       "dappnode.dnp.avatar": "avatar-url",
       "dappnode.dnp.origin": "origin-url",
-      "dappnode.dnp.chain": '{"driverd": "ethereum2-beacon-chain-prysm"}',
+      "dappnode.dnp.chain": '{"driverd": "ethereudm2-beacon-chain-prysm"}',
```

Comment:
Typo, must be `ethereum2` not `ethereudm2`

Reply:
It is an intended typo to force another situation for the test

Reply:
ah ok, thats why i just did a PR to your branch and didnt merge after it passed tests.

Reply:
Then please add a comment above this line